scripts/utils.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+check_user() {
+    if [ -z "$GITHUB_USER" ]; then
+        echo "Set GITHUB_USER"
+        exit
+    fi
+}
+
+check_initialization_options() {
```

Comment:
What's this one for?

Reply:
oops, thanks! this was for checking whether snapshot/timestamp/delegation keys etc were set, but it didn't make sense to use since only 1 script needed it. removing.